pkg/apis/serving/v1alpha1/service_validation.go

```diff
@@ -86,6 +86,9 @@ func (m *ManualType) Validate() *apis.FieldError {
 	return nil
 }
 
+// See #2819.
+const releaseLastRevisionKeyword = "@latestRevision"
```

Comment:
Is there somewhere you can put this to make this a public constant?

Last == Latest for consistency

```suggestion
const releaseLatestRevisionKeyword = "@latestRevision"
```

Reply:
When I gave suggestions like this, Matt had some reservation. So I didn't do public constant.
As for docs: will do. I noted that in the pr description.

Reply:
Hmm.. I would be curious on the reservation. We currently have similar string constants in [revision_types.go](https://github.com/knative/serving/blob/d92cc73ba14ba7f7ffad30256b77891914dc40be/pkg/apis/serving/v1alpha1/revision_types.go#L125) that get used as special values in the API.

[service_types.go](https://github.com/knative/serving/blob/d92cc73ba14ba7f7ffad30256b77891914dc40be/pkg/apis/serving/v1alpha1/service_types.go) seems like a reasonable place to define this for discoverability.